langlib/langlib-test/src/test/java/org/ballerinalang/langlib/test/statements/foreach/ForeachStringTypedBindingPatternsTests.java

```diff
@@ -25,12 +26,14 @@
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
+import static org.testng.Assert.assertEquals;
+
 /**
  * Tests for typed binding patterns in foreach.
  *
  * @since 0.985.0
  */
-@Test(enabled = false)
+@Test(enabled = true)
```

Comment:
default value of `enabled` is true, so you can remove the parameter, without explicitly making them true